### ISSUE TITLE
Code Simplicity: As a dev, I want terser, more memorable abstractions for character obj obtaining.

### ISSUE THREAD

**Issue by tchalvak** [BitLucid/ninjawars#761]
Things like:

     if ((int) RequestWrapper::getPostOrGet('target_id')) {
                $recipient = Player::find(RequestWrapper::getPostOrGet('target_id'))

Need a total rethinking.  Or, ya know, just controller parameters.

**Comment by tchalvak** [comment]
Dependent on #786

**Comment by tchalvak** [comment]
I think this will be nicely resolved by method parameters, so closing in favor of that. Wontfix
